scripts/systemtap/kem/kemd.c

```diff
@@ -162,24 +161,24 @@ static int kemd_init_module(void)
 		return -ENOMEM;
 	}
 	err = kemd_rbs_init(kemd_rbs);
-	if(err != 0) {
+	if (err != 0) {
 		printk(KERN_WARNING "Can't allocate kemd_rbs.\n");
 		goto out_rbs;
 	}
 	kemd_devices = kzalloc(kemd_nr_cpus * sizeof(*kemd_devices),
-			       GFP_KERNEL);
+                           GFP_KERNEL);
 	if (kemd_devices == NULL) {
 		printk(KERN_WARNING "Can't allocate mem for kemd_devices.\n");
 		goto out_rbs_inited;
 	}
 
 	if (kemd_major) {
 		devno = MKDEV(kemd_major, kemd_minor);
-		err = register_chrdev_region(devno, kemd_nr_cpus,
-					     KEMD_DEV_NAME);
+        err = register_chrdev_region(devno, kemd_nr_cpus,
+                                     KEMD_DEV_NAME);
 	} else {
-		err = alloc_chrdev_region(&devno, kemd_minor,
-					  kemd_nr_cpus, KEMD_DEV_NAME);
+        err = alloc_chrdev_region(&devno, kemd_minor,
+                                  kemd_nr_cpus, KEMD_DEV_NAME);
```

Comment:
Apparently, the new code does not comply with the indentation.
The original code is fine. Restore this.